backend/spec/factories/users.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :user do
+    username { "book_user" }
```

Comment:
There's a library called faker. In case you want fake values for stuff (names, addresses, url, tokens, and more)